### ISSUE TITLE
Problem with path argument

### ISSUE THREAD

**Issue by kingferiol** [duartemendes/aws-sam-api-proxy#2]
Hi! First of all super great project!

Just doing some tentative but I have an issue with the path

> sam-proxy start new-project-api --port 3000

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:112:11)
    at join (path.js:1040:7)
    at /usr/local/lib/node_modules/aws-sam-api-proxy/dist/serverlessFunctions/parseFromTemplate.js:81:32
    at Array.reduce (<anonymous>)
    at _default (/usr/local/lib/node_modules/aws-sam-api-proxy/dist/serverlessFunctions/parseFromTemplate.js:47:87)
    at _default (/usr/local/lib/node_modules/aws-sam-api-proxy/dist/index.js:71:73)
    at async Command.<anonymous> (/usr/local/lib/node_modules/aws-sam-api-proxy/dist/cli.js:37:3)
```

**Comment by duartemendes** [comment]
Hi, thank you!

Can you let me know what value are you using for `CodeUri` at the function resource or at the global level on your template file please?

**Comment by kingferiol** [comment]
At this time (very initial DEV) I didn't define that variable. I'm working on some Rest API and I just need a way to run few endpoints locally

**Comment by duartemendes** [comment]
Are you using InlineCode instead?

**Comment by kingferiol** [comment]
Yes, I am. Adding CodeUri '.' 
I was able to start everything but I'm not able to forward my request to lambda.
If you have any good suggestion I'm more then happy to accommodate

Thanks again for the help!

**Comment by duartemendes** [comment]
Well, at the moment this tool doesn't support InlineCode.

The workaround, that you'll eventually have to do, is to create a src folder and create a file for each of your lambda functions code. This way you get rid of InlineCode.

Then, your lambda function resource will look like this:

```yaml
CodeUri: ./src
Handler: GetResourceHandler.default
```

**Comment by kingferiol** [comment]
Finally, the execution of that command has been successfully completed

sam-proxy start my-api --port 3000 --env-vars ./src/test/resources/test_environment.json
Pulling required docker images, this might take a while... [ 'lambci/lambda:java8' ]
Found 0 containers, removing...
Pulling from lambci/lambda
Digest: sha256:c3696f62499e843defce73cbe96e3e314a6c222335b230ce4e7273793e95b253
Status: Image is up to date for lambci/lambda:java8
All required docker images have been pulled successfully.
Starting container {
  id: 'd1830c9ebe1b75d5e304192f89c5ba2efe02913c3e1b01beaedc7341cbfcf215',
  name: 'my_api_function_lambda'
}
Server is running on port 3000
The following endpoints are exposed:

│ (index) │ method  │ path                         │
| ------- | -------- | ------------------- |
│  0          │  'ANY'    │  '/my-api/{proxy+}'  │

But in the end, I got always the same answer

> {"status":"error","message":"Failed to find a function event for this request"}

The issue could be due to the fact that I have a single function that acts as a proxy

  MyFunction:
    Type: AWS::Serverless::Function
    Properties:
      Runtime: java8
      CodeUri: .
      Handler: com.MyHandler::handleRequest
      Events:
        CreateExecution:
          Type: Api
          Properties:
            Path: /my-api/{proxy+}
            Method: any

**Comment by duartemendes** [comment]
Yes, you're correct about the issue. I didn't take in consideration the ANY method. This should be easy to fix, I'll take a look at this today.

**Comment by kingferiol** [comment]
Yes I can confirm! Using just GET it works.

Another small issue that I was able to observe is that the request.getRequestContext() is null. This lead to some failures.

**Comment by duartemendes** [comment]
@kingferiol fix for the ANY method has been published under version 0.0.6. Please update your CLI and retry.

Regarding requestContext, correct, I didn't yet add support for it, if it's something you need feel free to open another issue and I'll take a look to understand what's the effort required to support it.

Cheers, closing this issue.

**Comment by kingferiol** [comment]
Super! Thanks a lot!
